### DISCUSSION TITLE
How do I do a silent install through winget?

### DISCUSSION THREAD

**Discussion by DuckDuckStudio** [neon-nyan/Collapse#649] [Q&A] [answered]
In [microsoft/winget-pkgs#202767](https://github.com/microsoft/winget-pkgs/pull/202767#issuecomment-2563926599) , the winget-pkgs maintainer noticed that the silent install parameter in the updated manifest seemed incorrect, and the prompt message told he to try the `--help` parameter, but there was no prompt message or output after execution.

I'm wondering if there is a way to perform a silent install of the program for versions in this PR and newer? And how do I find the output of the `--help` or similar help commands?

**Comment by bagusnl** [comment, marked as answer]
Heya! 

Velopack docs site seems to be quite lacking atm but scouring thru their Discord server seems that `--silent` should do the trick, albeit I havent tried myself
